clusters/app.ci/quayio-pull-through-cache/quayio-pull-through-cache-us-east-1.yaml

```diff
@@ -1,6 +1,38 @@
 apiVersion: v1
 kind: List
 items:
+- kind: ConfigMap
+  apiVersion: v1
+  data:
+    config.yml: |-
+      version: 0.1
+      log:
+        fields:
+          service: registry
+      storage:
+        s3:
+          encrypt: true
+          rootdirectory: /quay.io
+          region: us-east-1
+          bucket: quayio-pull-through-cache-us-east-1
```

Comment:
hmm so every config map will repeat all the fields because we want specific bucket hardcoded? How this is better than variable?

Reply:
This one is more of a choice not an obvious improvement, I admit.
Here is what is in my mind.
Each deployment has its own config file from its own configMap not from a file from a build image so that we do not need to rebuilt the image for a new configuration file.
The env. var. is only for the credentials that I cannot put here in the configMap as plain text.

Reply:
It's maybe true but I do not see a case where we would like to have different configurations per different zone. It should be the same with the exception of zone name and bucket. In the future, when knowledge about this work is kind of forgotten, I am afraid that somebody will come, edit only one configuration and forget about the other. This is redundancy and as in programming we tend to remove redundant code and extract common pieces, I would think the same should be the configuration. This is my point of view at least.